index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="ro">
+    <meta charset="UTF-8">
+    
+    <head class="header">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title> Save the planet</title>
+        <link href="./CSS/homestyle.css" rel=stylesheet>         
+    </head>
+    <body>
+        <section class="coverContainer">
+            <div class="coverImage">
+                <div class="shape"><img  src="./Photos/Shape.png" alt="shape"></div>
```

Comment:
alt-ul ar trebui sa fie foarte sugestiv "Photo of ... " si detalii despre ce in poza, 4-5 cuvinte (ideal)